pandas/io/parsers.py

```diff
@@ -2153,8 +2155,12 @@ def _try_convert_dates(parser, colspec, data_dict, columns):
 
     new_name = '_'.join([str(x) for x in colnames])
     to_parse = [data_dict[c] for c in colnames if c in data_dict]
-
     new_col = parser(*to_parse)
+    if hasattr(new_col, 'dtype') and any(r in str(new_col.dtype) for r in
```

Comment:
I don't know how to "rebase" the master branch. I tried ``git rebase -i`` on master, and it just said ``noop`` where the commit list should be.